app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue

```diff
@@ -13,7 +13,7 @@
       :key="account.id"
       class="account-selector"
     >
-      <a :href="`/app/accounts/${account.id}/dashboard`">
+      <a @click="onChangeAccount(account.id)">
```

Comment:
Let's use a woot button with variant link here

Reply:
@nithindavid The `woot-button` doesn't support the multiple labels. Should I update the button component ? 
![Screenshot 2022-09-14 at 12 04 31 PM](https://user-images.githubusercontent.com/12408980/190077811-8fe78390-cd47-41ce-a53b-032f49c14f39.png)

Reply:
Or use a normal HTML button. Since `a` tag without href will throw an invalid ARAI error

Reply:
@nithindavid Done

Reply:
@nithindavid Could you please review the changes?